src/Domain/CoordinateMaps/FocallyLiftedFlatSide.cpp

```diff
@@ -0,0 +1,290 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Domain/CoordinateMaps/FocallyLiftedFlatSide.hpp"
+
+#include <cmath>
+#include <limits>
```

Comment:
include `<array>`,  `<type_traits>`, `<optional>`, `Utilities/Gsl.hpp`, `Utilities/ErrorHandling/Assert.hpp`, `Utilities/TypeTraits/RemoveReferenceWrapper.hpp`

Reply:
I wish we had a working include-what-you-use...